### PR TITLE
[Testing ]cuda11 binary test

### PR DESCRIPTION
**Pull request by mszhanyi**
Don't Review
Fixes #{issue number}
Testing https://github.com/pytorch/builder/pull/564

### CODE REVIEW

File: .circleci/scripts/binary_checkout.sh [hunk 1]
```diff
@@ -34,11 +34,13 @@ else
 fi
 
 # Clone the Pytorch branch
-retry git clone https://github.com/pytorch/pytorch.git "$PYTORCH_ROOT"
-pushd "$PYTORCH_ROOT"
+retry git clone https://github.com/pytorch/builder.git "$BUILDER_ROOT"
```
Comment (peterjc123) [inline .circleci/scripts/binary_checkout.sh:37]: LOL. Why do you change like this? Isn't it easier to just change line 59 into the following line?
```bash
retry git clone -q https://github.com/mszhanyi/builder.git -b zhanyi/addcu11bintest "$PYTORCH_ROOT"
```

File: .circleci/cimodel/data/simple/binary_smoketest.py [hunk 2]
```diff
@@ -153,7 +160,17 @@ def gen_tree(self):
             "executor": "windows-with-nvidia-gpu",
         },
     ),
-
+    SmoketestJob(
```
Comment (peterjc123) [inline .circleci/cimodel/data/simple/binary_smoketest.py:163]: Actually we have smoke test jobs for nightly packages, so this is just for testing, right?

Comment (mszhanyi) [reply .circleci/cimodel/data/simple/binary_smoketest.py:163]: Yes, what I want is to make sure my PR in builder can run properly.
And I can get the [verified link](https://app.circleci.com/pipelines/github/pytorch/pytorch/235594/workflows/ad93d2da-ccba-4dad-b835-fd1d6b3023f5/jobs/8725700) easily

File: .circleci/scripts/binary_checkout.sh (hunk 1 above)
Comment (mszhanyi) [reply .circleci/scripts/binary_checkout.sh:37]: Sorry, at the begining, I just want to run a test quickly so didn't take time to walk through the whole script

**Pull request closed without merging**